daemon/logger/jsonfilelog/jsonlog/fuzz_test.go

```diff
@@ -0,0 +1,21 @@
+package jsonlog
+
+import (
+	"bytes"
+	"testing"
+
+	fuzz "github.com/AdaLogics/go-fuzz-headers"
+)
+
+func FuzzJSONLogsMarshalJSONBuf(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		ff := fuzz.NewConsumer(data)
+		l := &JSONLogs{}
+		err := ff.GenerateStruct(l)
+		if err != nil {
+			return
```

Comment:
No. If we cannot create a `JSONLogs` struct, it is because the test case was insufficient, and we should just return gracefully and let the fuzzer try again.
For example, if the test case is "00", then there will not be enough bytes to add values to the struct, but it is not an error.

Reply:
Ah, thanks; makes sense 👍